assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/mini-cart-products-table-block/block.json

```
@@ -0,0 +1,26 @@
+{
+	"name": "woocommerce/mini-cart-products-table-block",
+	"version": "1.0.0",
+	"title": "Mini Cart Products Table",
+	"description": "Block that display the products table of the Mini Cart block.",
```

Same: :slightly_smiling_face: 
```suggestion
	"description": "Block that displays the products table of the Mini Cart block.",
```